tests/test_enterprise/api/test_views.py

```diff
@@ -787,6 +795,94 @@ def test_enterprise_customer_catalogs_detail_pagination(self, mock_catalog_api_c
         assert response == expected_result
 
     @ddt.data(
+        (False, {}, {}, None),
+        (False, {}, {'course_run_ids': ['fake1', 'fake2']}, None),
+        (
+            True,
+            {
+                'key': [
+                    fake_catalog_api.FAKE_COURSE_RUN['key'],
+                    fake_catalog_api.FAKE_COURSE_RUN2['key']
+                ]
+            },
+            {
+                'course_run_ids': [
+                    fake_catalog_api.FAKE_COURSE_RUN['key'],
+                    fake_catalog_api.FAKE_COURSE_RUN2['key']
+                ]
+            },
+            None
+        ),
+        (
+            True,
+            {
+                'uuid': [
+                    fake_catalog_api.FAKE_PROGRAM_RESPONSE1['uuid'],
+                    fake_catalog_api.FAKE_PROGRAM_RESPONSE2['uuid']
+                ]
+            },
+            {
+                'program_uuids': [
+                    fake_catalog_api.FAKE_PROGRAM_RESPONSE1['uuid'],
+                    fake_catalog_api.FAKE_PROGRAM_RESPONSE2['uuid']
+                ]
+            },
+            None
+        ),
+        (
+            True,
+            {},
+            {
+                'course_run_ids': [
+                    fake_catalog_api.FAKE_COURSE_RUN['key'],
+                    fake_catalog_api.FAKE_COURSE_RUN2['key']
+                ]
+            },
+            [fake_catalog_api.FAKE_COURSE_RUN, fake_catalog_api.FAKE_COURSE_RUN2]
+        ),
+        (
+            True,
+            {},
+            {
+                'program_uuids': [
+                    fake_catalog_api.FAKE_PROGRAM_RESPONSE1['uuid'],
+                    fake_catalog_api.FAKE_PROGRAM_RESPONSE2['uuid']
+                ]
+            },
+            [fake_catalog_api.FAKE_PROGRAM_RESPONSE1, fake_catalog_api.FAKE_PROGRAM_RESPONSE2]
+        ),
+    )
+    @ddt.unpack
+    @mock.patch('enterprise.models.CourseCatalogApiServiceClient')
+    def test_enterprise_catalog_contains_content(self, contains_content, content_filter, query_params, search_results,
+                                                 mock_catalog_api_client):
+        """
+        The ``programs`` detail endpoint should return correct results from course discovery,
+        with enterprise context in courses.
```

Comment:
@douglashall Copy-paste leftover? This doesn't seem to describe what the test does :)

Reply:
Yup.